.env.example

```
@@ -1,43 +1,11 @@
-# API key for authenticating inbound Alpaca requests (REQUIRED, min 32 chars)
-# Generate with: openssl rand -hex 32
 ISSUER_API_KEY=${ISSUER_API_KEY}
-
-# Optional: IP whitelist for issuer endpoints (comma-separated CIDR notation)
-# If not set or commented out, IP whitelisting is DISABLED (not recommended for production)
-# Uncomment and set when you receive Alpaca's IP ranges:
-# ALPACA_IP_RANGES=1.2.3.0/24,5.6.7.8/32
-
-# Alpaca API configuration (required for ITN integration)
-ALPACA_API_BASE_URL=https://broker-api.alpaca.markets
+ALPACA_IP_RANGES=${ALPACA_IP_RANGES}
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ALPACA_IP_RANGES key should go before the BOT_WALLET key

(UnorderedKey)